pkg/api/avatar/avatar.go

```diff
@@ -51,7 +51,10 @@ func HashEmail(email string) string {
 	email = strings.ToLower(email)
 
 	h := md5.New()
-	h.Write([]byte(email))
+	if _, err := h.Write([]byte(email)); err != nil {
+		// TODO: Deal with error
+		log.Warn("Failed to hash email address: %s", err)
```

Comment:
Need to be `Warnf` to work. But use argument `log.Warn("...", "err", err` instead.

Reply:
@marefr See my above comment.

Reply:
`HashEmail` function doesn't seem to be in use anymore so should be able to remove it, but please verify